contrib/systemd/spacenavd.service

```diff
@@ -1,12 +1,10 @@
 [Unit]
 Description=3Dconnexion Input Devices Userspace Driver
-After=syslog.target
```

Comment:
We are no longer using syslog, so we don't need to wait for the syslog daemon to start.